sql-training/src/ex07.sql

```diff
@@ -0,0 +1,8 @@
+SELECT STRING_AGG (m.title, ';') as Brand, (m.id) id, pin.data_source_uid, STRING_AGG(ins.username, ';') as insta_user
+  FROM movies m
+  LEFT JOIN instagram_accounts ins on ins.movie_id = m.id

```

Comment:
What will be the result if you use inner join conditions here?

Reply:
The same result will come